<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Preferences</string>
    </property>
    <widget class="QMenu" name="menuSensitivity">
     <property name="title">
      <string>Sensitivity</string>
     </property>
     <addaction name="actionLow"/>
     <addaction name="actionMedium"/>
     <addaction name="actionHigh"/>
    </widget>
    <addaction name="menuSensitivity"/>
    <addaction name="separator"/>
    <addaction name="actionUse_Color_to_Distinguist"/>
    <addaction name="actionVerbose_Mode"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import Images...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export Current...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Low</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="actionHigh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>High</string>
   </property>
  </action>
  <action name="actionUse_Color_to_Distinguist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Color to Distinguish</string>
   </property>
  </action>
  <action name="actionStart_Analyze">
   <property name="text">
    <string>Start Analyze</string>
   </property>
  </action>
  <action name="actionRest">
   <property name="text">
    <string>Rest</string>
   </property>
  </action>
  <action name="actionVerbose_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Verbose Mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
